src/torchmetrics/classification/negative_predictive_value.py

```diff
@@ -0,0 +1,516 @@
+    - ``target`` (:class:`~torch.Tensor`): An int tensor of shape ``(N, ...)``
+
+    As output to ``forward`` and ``compute`` the metric returns the following output:
+
+    - ``bs`` (:class:`~torch.Tensor`): If ``multidim_average`` is set to ``global``, the metric returns a scalar value.
+      If ``multidim_average`` is set to ``samplewise``, the metric returns ``(N,)`` vector consisting of a scalar value
+      per sample.
+
+    If ``multidim_average`` is set to ``samplewise`` we expect at least one additional dimension ``...`` to be present,
+    which the reduction will then be applied over instead of the sample dimension ``N``.
+
+    Args:
+        threshold: Threshold for transforming probability to binary {0,1} predictions
+        multidim_average:
+            Defines how additionally dimensions ``...`` should be handled. Should be one of the following:
+
+            - ``global``: Additional dimensions are flatted along the batch dimension
+            - ``samplewise``: Statistic will be calculated independently for each sample on the ``N`` axis.
+              The statistics in this case are calculated over the additional dimensions.
+
+        ignore_index:
+            Specifies a target value that is ignored and does not contribute to the metric calculation
+        validate_args: bool indicating if input arguments and tensors should be validated for correctness.
+            Set to ``False`` for faster computations.
+
+    Example (preds is int tensor):
+        >>> from torch import tensor
+        >>> from torchmetrics.classification import BinaryNegativePredictiveValue
+        >>> target = tensor([0, 1, 0, 1, 0, 1])
+        >>> preds = tensor([0, 0, 1, 1, 0, 1])
+        >>> metric = BinaryNegativePredictiveValue()
+        >>> metric(preds, target)
+        tensor(0.6667)
+
+    Example (preds is float tensor):
+        >>> from torchmetrics.classification import BinaryNegativePredictiveValue
+        >>> target = tensor([0, 1, 0, 1, 0, 1])
+        >>> preds = tensor([0.11, 0.22, 0.84, 0.73, 0.33, 0.92])
+        >>> metric = BinaryNegativePredictiveValue()
+        >>> metric(preds, target)
+        tensor(0.6667)
+
+    Example (multidim tensors):
+        >>> from torchmetrics.classification import BinaryNegativePredictiveValue
+        >>> target = tensor([[[0, 1], [1, 0], [0, 1]], [[1, 1], [0, 0], [1, 0]]])
+        >>> preds = tensor([[[0.59, 0.91], [0.91, 0.99],  [0.63, 0.04]],
+        ...                 [[0.38, 0.04], [0.86, 0.780], [0.45, 0.37]]])
+        >>> metric = BinaryNegativePredictiveValue(multidim_average='samplewise')
+        >>> metric(preds, target)
+        tensor([0.0000, 0.3333])
+
+    """
+
+    plot_lower_bound: float = 0.0
+    plot_upper_bound: float = 1.0
+
+    def compute(self) -> Tensor:
+        """Compute metric."""
+        tp, fp, tn, fn = self._final_state()
+        return _specificity_reduce(tp, fp, tn, fn, average="binary", multidim_average=self.multidim_average)
```

Comment:
Shouldn't this be _negative_predictive_value_reduce?

Reply:
![bug](https://github.com/Lightning-AI/torchmetrics/assets/105210894/72ee63a4-be28-40e5-920f-8a68484bd1a9)
There's something wrong here.
You may need to redefine a new function: _negative_predictive_value_reduce.
NPV = TN / (TN + FN)

Reply:
_negative_predictive_value_reduce has been defined [here](https://github.com/Lightning-AI/torchmetrics/blob/32e35d1d0450d01b6269e09a4e1611973851d250/src/torchmetrics/functional/classification/negative_predictive_value.py#L37), but isn't used on this line for some reason